packages/@dcl/react-ecs/src/components/Label/types.ts

```diff
@@ -3,4 +3,27 @@ import { PBUiText } from '@dcl/ecs'
 /**
  * @public
  */
-export type UiTextProps = PBUiText
+export type UiLabelProps = Omit<PBUiText, 'textAlign' | 'font'> & {
+  /** default='middle-center' */
+  textAlign?: TextAlign | undefined
+  /** default='sans-serif' */
+  font?: UiFont | undefined
+}
+
+/**
+ * @public
+ */
+export type UiFont = 'sans-serif' | 'serif' | 'monospace'
+/**
+ * @public
+ */
+export type TextAlign =
```

Comment:
I think we are going to have trouble with this typings names unless we change the names to very specific ones like "TextAlignType"

Reply:
Or use namespaces